.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: Cargo Build & Test
```

Comment:
Did you expect this file to run in the PR?

Reply:
It needs to be merged to affect PRs. So far it runs on my repo: https://github.com/rasendubi/eppo-rust-sdk/actions/runs/8803705195/job/24162446849